src/ctia/entity/asset_properties.clj

```diff
@@ -0,0 +1,134 @@
+(ns ctia.entity.asset-properties
+  (:require [ctia.domain.entities :refer [default-realize-fn]]
+            [ctia.schemas.core :refer [def-acl-schema def-stored-schema]]
+            [ctia.schemas.utils :as csu]
+            [ctia.schemas.sorting :as sorting]
+            [ctia.stores.es.mapping :as em]
+            [ctia.stores.es.store :refer [def-es-store]]
+            [ctim.schemas.asset-properties :as asset-properties-schema]
+            [schema-tools.core :as st]
+            [ctia.http.routes.crud :refer [entity-crud-routes]]
+            [ctia.http.routes.common :as routes.common]
+            [flanders.utils :as fu]
+            [schema.core :as s]))
+
+(def-acl-schema AssetProperties
+  asset-properties-schema/AssetProperties
+  "asset-properties")
+
+(def-acl-schema PartialAssetProperties
+  (fu/optionalize-all asset-properties-schema/AssetProperties)
+  "partial-asset-properties")
+
+(s/defschema PartialAssetPropertiesList
+  [PartialAssetProperties])
+
+(def-acl-schema NewAssetProperties
+  asset-properties-schema/NewAssetProperties
+  "new-asset-properties")
+
+(def-stored-schema StoredAssetProperties AssetProperties)
+
+(s/defschema PartialStoredAssetProperties
+  (csu/optional-keys-schema StoredAssetProperties))
+
+(def realize-asset-properties
+  (default-realize-fn "asset-properties" NewAssetProperties StoredAssetProperties))
+
+(def asset-properties-mapping
+  {"asset-properties"
+   {:dynamic false
+    :properties
+    (merge
+     em/base-entity-mapping
+     em/sourcable-entity-mapping
+     em/stored-entity-mapping
+     {:identity    em/tg-identity
```

Comment:
This property is not defined in the schema: https://github.com/threatgrid/ctim/pull/293/files
In case it should be added to the schema (I doubt it), [here](https://github.com/threatgrid/ctia/blob/master/src/ctia/stores/es/mapping.clj#L151) is the mapping you are referring to.

Reply:
Removed. Not needed here.